core/domain/feedback_services.py

```diff
@@ -143,26 +191,50 @@ def _get_message_from_model(message_model):
 
 
 def get_messages(exploration_id, thread_id):
+    """Fetch all messages of the given thread indexed by
+        [exploration_id].[thread_id]
+
+    Args:
+        exploration_id: str.
+        thread_id: str.
+
+    Returns:
+        list of FeedbackMessage. Contains all the messages.
+    """
     return [
         _get_message_from_model(m)
         for m in feedback_models.FeedbackMessageModel.get_messages(
             exploration_id, thread_id)]
 
 
 def get_message(exploration_id, thread_id, message_id):
+    """Fetch the message indexed by [exploration_id].[thread_id].[message_id]
+
+    Args:
+        exploration_id: str.
+        thread_id: str.
+        message_id: int.
```

Comment:
According to the model layer:

    # 0-based sequential numerical ID. Sorting by this field will create the
    # thread in chronological order.
    message_id = ndb.IntegerProperty(required=True, indexed=True)

So I suppose it's an integer?